kANe/v5p1/1_Vratas_and_Utsavas/03_vratAdhikArin_and_related_topics.md

```diff
@@ -71,83 +71,81 @@ Purpose of most vratas
 
 55
 
-Another remarkable matter is that the Mahābhārata provides that brāhmaṇas and ksatriyas should not engage in a continuous fast for more than three days and that vaiśyas and $ūdras can observe a continuous fast for two days only and a fast for three days is not laid down for these two classes by those who know dharma. Devala also has a similar provision, 140
+Another remarkable matter is that the Mahābhārata provides that brāhmaṇas and kśatriyas should not engage in a continuous fast for more than three days and that vaiśyas and Śūdras can observe a continuous fast for two days only and a fast for three days is not laid down for these two classes by those who know _dharma_. Devala also has a similar provision[^140].
```

Comment:
kṣatriyas

Reply:
In IAST mode, for letter 's',  I get only 'ś' and not 'ṣ'. 
Please tell me how to get 'ṣ'

Reply:
Try S or shh

Reply:
I am not getting it with any combination.

Reply:
corrected